backend/src/controllers/link.controller.js

```
@@ -132,6 +136,9 @@
   async getLinksByHelperId(req, res) {
     try {
       const { helperId } = req.query;
+      if (!helperId || isNaN(helperId)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)